docs/security/localization-threat.rst

```diff
@@ -0,0 +1,131 @@
+
+3. Repudiation
+++++++++++++++
+
+- **R1. Disputes over malicious or poor-quality translations.**
+
+  - Risk: Translators deny responsibility for injected issues.
+  - Mitigations:
+
+    - All changes in Weblate are logged.
+    - Use Weblate with version control for immutable history.
+
+4. Information Disclosure
++++++++++++++++++++++++++
+
+- **I1. Leakage of unreleased product details.**
+
+  - Risk: Translators gain early access to unreleased features or confidential terminology.
+  - Mitigations:
+
+    - Segment projects to limit access to sensitive strings.
+    - Apply non-disclosure agreements with external agencies.
+    - Delay translation of highly confidential strings until public release.
+
+- **I2. Exposure of personal data within strings.**
+
+  - Risk: Translators might access or misuse embedded user data.
+  - Mitigations:
+
+    - Avoid exposing real user data in source strings.
+    - Use placeholders for sensitive fields.
+
+5. Denial of Service
+++++++++++++++++++++
+
+- **D1. Bulk submission of junk translations.**
+
+  - Risk: Review queues overwhelmed; release timelines disrupted.
+  - Mitigations:
+
+    - Choose an appropriate workflow to match your team capacity. :ref:`workflow-customization` can allow you to tweak this on a language basis.
+    - Configure automated translation quality checks; see :ref:`checks`.
+
+6. Elevation of Privilege
++++++++++++++++++++++++++
+
+- **E1. The translator gains unauthorized project-wide or administrative rights.**
+
+  - Risk: Escalation leading to tampering or data exposure.
+  - Mitigations:
+
+    - Apply the principle of least privilege.
+    - Regularly review access rights and group memberships.
+
+Asset Inventory
+---------------
+
+- **Source Strings:** May contain unreleased product features or legal text.
+- **Translated Strings:** Output presented directly to end users.
+- **User Data Placeholders:** Names, emails, or IDs referenced in strings.
+- **Access Credentials:** Accounts for translators, agencies, or bots.
+
+Trust Boundaries
+----------------
+
+- **Organization ↔ Translators:** Authentication and role-based access must be enforced.
+- **Translation Platform ↔ Source Control:** Synchronization requires secured tokens/keys.
+- **Translators ↔ Translation Platform:** All input must be sanitized before integration into builds.
+- **Platform ↔ End Users:** Ensures that translations cannot be weaponized for code injection.
+
+Mitigation Summary
+------------------
+
+- Enforce 2FA and RBAC for translator accounts; see :ref:`project-enforced_2fa`.
+- Require non-disclosure agreements or contracts for professional translators.
+- Use automated quality/security scanning for translations; see :ref:`checks`.
+- Perform peer review of critical strings; see :ref:`peer-review` or :ref:`reviews`.
+- Limit project visibility to reduce exposure of sensitive content; see :ref:`acl`.
+- Regularly patch and harden your Weblate.
```

Comment:
This mostly makes sense to me in context, but could still be worth clarifying: what do you mean by "your Weblate"? Do you mean my Weblate configuration? I'm not 100% clear on what you're suggesting here.